examples/Microsoft.Compute/add UAI to VirtualMachineScaleSets created by AKS/main.tf

```diff
@@ -0,0 +1,63 @@
+terraform {
```

Comment:
Can we avoid using empty space in the directory name (same for the other places)?